compel/arch/x86/src/lib/include/uapi/asm/cpu.h

```diff
@@ -15,334 +15,334 @@
+#define X86_FEATURE_PAE (0 * 32 + 6) /* Physical Address Extensions */
+#define X86_FEATURE_MCE (0 * 32 + 7) /* Machine Check Exception */
+#define X86_FEATURE_CX8 (0 * 32 + 8) /* CMPXCHG8 instruction */
+#define X86_FEATURE_APIC (0 * 32 + 9) /* Onboard APIC */
+#define X86_FEATURE_SEP (0 * 32 + 11) /* SYSENTER/SYSEXIT */
+#define X86_FEATURE_MTRR (0 * 32 + 12) /* Memory Type Range Registers */
+#define X86_FEATURE_PGE (0 * 32 + 13) /* Page Global Enable */
+#define X86_FEATURE_MCA (0 * 32 + 14) /* Machine Check Architecture */
+#define X86_FEATURE_CMOV (0 * 32 + 15) /* CMOV instructions (plus FCMOVcc, FCOMI with FPU) */
+#define X86_FEATURE_PAT (0 * 32 + 16) /* Page Attribute Table */
+#define X86_FEATURE_PSE36 (0 * 32 + 17) /* 36-bit PSEs */
+#define X86_FEATURE_PN (0 * 32 + 18) /* Processor serial number */
+#define X86_FEATURE_CLFLUSH (0 * 32 + 19) /* CLFLUSH instruction */
+#define X86_FEATURE_DS (0 * 32 + 21) /* "dts" Debug Store */
+#define X86_FEATURE_ACPI (0 * 32 + 22) /* ACPI via MSR */
+#define X86_FEATURE_MMX (0 * 32 + 23) /* Multimedia Extensions */
+#define X86_FEATURE_FXSR (0 * 32 + 24) /* FXSAVE/FXRSTOR, CR4.OSFXSR */
+#define X86_FEATURE_XMM (0 * 32 + 25) /* "sse" */
+#define X86_FEATURE_XMM2 (0 * 32 + 26) /* "sse2" */
+#define X86_FEATURE_SELFSNOOP (0 * 32 + 27) /* "ss" CPU self snoop */
+#define X86_FEATURE_HT (0 * 32 + 28) /* Hyper-Threading */
+#define X86_FEATURE_ACC (0 * 32 + 29) /* "tm" Automatic clock control */
+#define X86_FEATURE_IA64 (0 * 32 + 30) /* IA-64 processor */
+#define X86_FEATURE_PBE (0 * 32 + 31) /* Pending Break Enable */
 
 /* AMD-defined CPU features, CPUID level 0x80000001, word 1 */
 /* Don't duplicate feature flags which are redundant with Intel! */
-#define X86_FEATURE_SYSCALL		(1*32+11) /* SYSCALL/SYSRET */
-#define X86_FEATURE_MP			(1*32+19) /* MP Capable */
-#define X86_FEATURE_NX			(1*32+20) /* Execute Disable */
-#define X86_FEATURE_MMXEXT		(1*32+22) /* AMD MMX extensions */
-#define X86_FEATURE_FXSR_OPT		(1*32+25) /* FXSAVE/FXRSTOR optimizations */
-#define X86_FEATURE_GBPAGES		(1*32+26) /* "pdpe1gb" GB pages */
-#define X86_FEATURE_RDTSCP		(1*32+27) /* RDTSCP */
-#define X86_FEATURE_LM			(1*32+29) /* Long Mode (x86-64, 64-bit support) */
-#define X86_FEATURE_3DNOWEXT		(1*32+30) /* AMD 3DNow extensions */
-#define X86_FEATURE_3DNOW		(1*32+31) /* 3DNow */
+#define X86_FEATURE_SYSCALL (1 * 32 + 11) /* SYSCALL/SYSRET */
+#define X86_FEATURE_MP (1 * 32 + 19) /* MP Capable */
+#define X86_FEATURE_NX (1 * 32 + 20) /* Execute Disable */
+#define X86_FEATURE_MMXEXT (1 * 32 + 22) /* AMD MMX extensions */
+#define X86_FEATURE_FXSR_OPT (1 * 32 + 25) /* FXSAVE/FXRSTOR optimizations */
+#define X86_FEATURE_GBPAGES (1 * 32 + 26) /* "pdpe1gb" GB pages */
+#define X86_FEATURE_RDTSCP (1 * 32 + 27) /* RDTSCP */
+#define X86_FEATURE_LM (1 * 32 + 29) /* Long Mode (x86-64, 64-bit support) */
+#define X86_FEATURE_3DNOWEXT (1 * 32 + 30) /* AMD 3DNow extensions */
+#define X86_FEATURE_3DNOW (1 * 32 + 31) /* 3DNow */
```

Comment:
Not sure about this one, do we really want to enforce no human readable aligned neighbor lines (for macro values or comments or something else)?

Can we use `AlignConsecutiveMacros: true`?

```
--- a/scripts/fetch-clang-format.sh
+++ b/scripts/fetch-clang-format.sh
@@ -11,4 +11,5 @@ curl -s "${URL}" | sed -e "
        s,ColumnLimit: 80,ColumnLimit: 120,g;
        s,Intended for clang-format >= 4,Intended for clang-format >= 11,g;
        s,ForEachMacros:,ForEachMacros:\n  - 'for_each_pstree_item',g;
+       s,\(AlignTrailingComments:.*\)$,\1\nAlignConsecutiveMacros: true,g;
 "  > .clang-format
```

Reply:
I can add it. No Problem.